Source/Scene/Scene.js

```diff
@@ -948,6 +948,18 @@ define([
             }
         },
 
+        /**
+         * Returns <code>true</code> if specular environment maps are supported.
+         * @memberof Scene.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        specularEnvironmentMapSupported : {
```

Comment:
`specularEnvironmentMapsSupported` sounds better

Reply:
Also model the doc after `invertClassificationSupported`: there shouldn't be a default value and add a `@see` to `specularEnvironmentMaps`